students/bedkowska_julita/lesson_01_basics/apple_sharing.py

```diff
@@ -0,0 +1,5 @@
+n = int(input("input number of students: "))
+k = int(input("input number of apples: "))
```

Comment:
Try to name variables properly, like students or apples. No one know wkat 'k' or 'n' does it mean